improve enable suspect commits content (#5094)
getsentry/sentry-docs: src/docs/product/releases/suspect-commits/index.mdx

---
title: Suspect Commits
sidebar_order: 2
description: "Learn how associating commits to a release allows you to track suspect commits."
---

In your release process, you can add a step to create a release in Sentry and associate it with commits from your linked repository. When you've done this, you'll be able to see suspect commits and suggested assignees in issues, and resolve issues by commit, as described below.

If you're using one of our [release automation](/product/releases/setup/release-automation/) options or our [manual setup](/product/releases/setup/manual-setup-releases/) instructions, suspect commits should already be enabled. However, we've included directions for enabling suspect commits using the [Sentry CLI](#using-the-cli) and [our API](#using-the-api) at the end of this page, just in case you need them. 

## Suspect Commits & Suggested Assignees

When you've enabled suspect commits, we can tie together the commits in the release, including:

- Files touched by those commits
- Files observed in the stack trace
- Authors of those files
- The issue number in a commit message

> Suspect commits and suggested assignees will then display on the **Issue Details** page:

Author: ```suggestion
Suspect commits and suggested assignees will then display on the **Issue Details** page in [sentry.io](https://sentry.io):
```

![The suspect commits and suggested assignees for an issue](suspect-commits-highlighted.png)

## Resolve Issues by Commit

> Additionally, when you've enabled suspect commits, youll be able to resolve issues by including the issue ID in your commit message. You can find the issue ID at the top of the **Issue Details** page, next to the assignee dropdown. For example, a commit message might look like this:

Author: ```suggestion
Additionally, when you've enabled suspect commits, you'll be able to resolve issues by including the issue ID in your commit message. You can find the issue ID at the top of the **Issue Details** page, next to the assignee dropdown. 

A commit message might look like this, for example:
```

---
title: Suspect Commits
sidebar_order: 2

> description: "Learn how to associate commits to a release to track suspect commits."

Author: ```suggestion
description: "Learn how associating commits to a release allows you to track suspect commits."
```

---

> In your release process, you can add a step to create a release in Sentry and associate it with commits from your linked repository. When you've done this, you'll be able to:

Author: ```suggestion
In your release process, you can add a step to create a release in Sentry and associate it with commits from your linked repository. When you've done this, you'll be able to see suspect commits and suggested assignees in issues, and resolve issues by commit, as described below.
```

> - See suspect commits and suggested assignees in issues
> - Resolve issues by commit

Author: ```suggestion
```

> If you're using one of our [release automation](/product/releases/setup/release-automation/) options or our [manual setup](/product/releases/setup/manual-setup-releases/) instructions, suspect commits should already be enabled.

Author: ```suggestion
If you're using one of our [release automation](/product/releases/setup/release-automation/) options or our [manual setup](/product/releases/setup/manual-setup-releases/) instructions, suspect commits should already be enabled. However, we've included directions for enabling suspect commits using the [Sentry CLI](#using-the-cli) and [our API](#using-the-api) at the end of this page, just in case you need them. 
```

## Suspect Commits & Suggested Assignees

When you've enabled suspect commits, we can tie together the commits in the release, including:

- Files touched by those commits
- Files observed in the stack trace
- Authors of those files
- The issue number in a commit message

Suspect commits and suggested assignees will then display on the **Issue Details** page:

> ![](suspect-commits-highlighted.png)

Author: ```suggestion
![The suspect commits and suggested assignees for an issue](suspect-commits-highlighted.png)
```
